extensions/plugin-history-sync/src/historySyncPlugin.tsx

```diff
@@ -161,60 +161,57 @@ export function historySyncPlugin<T extends { [activityName: string]: any }>(
         }
 
         function resolvePath() {
-          if (context?.req?.path) {
-            return context.req.path;
+          if (context?.req?.path && typeof context.req.path === 'string') {
+            return context.req.path as string
           }
           if (isServer) {
             return null;
           }
 
           if (options.useHash) {
-            return window.location.hash.split("#")[1];
+            return window.location.hash.split("#")[1] ?? '/';
           } else {
             return window.location.pathname + window.location.search;
           }
```

Comment:
```suggestion

          return window.location.pathname + window.location.search;
```
일부러 else안에 넣어두신 건지 궁금해요~ eslint가 잡고 있어서용

Reply:
분기점을 기준으로 A/B 로 확실하게 나눈다는 것을 표현하려는 것인줄 알았는데, 지금보니 useHash 라는 분기는 여집합이 훨씬 커서 else 가 없어도 될 거 같네요

Reply:
반영 완료~